docs/configure-scalardb-for-your-database.md

```diff
@@ -0,0 +1,243 @@
+# Configure ScalarDB for Your Database
```

Comment:
@josh-wong @choplin I'm not sure this title is appropriate.

The goal of this document is to provide introductory steps for new users to get started with ScalarDB, but I don't think the goal is to explain how to configure ScalarDB. We already have a separate [document](https://github.com/scalar-labs/scalardb/blob/9f45c79c406b12d9be58ebc39d19ec27803352fd/docs/configurations.md) addressing that. So I think the title `Getting Started with ScalarDB` is more appropriate. What do you think?

Reply:
@brfrn169 I see your point. Since we have another document that describes how to configure a database in detail, I'm OK with changing the title of this document back to "Getting Started with ScalarDB." 

In addition, we also have other "Getting Started" docs for features in ScalarDB that are similar to this one. If we decide a different title is more suitable later, we can always change it.

Reply:
I also think `Getting Started with ScalarDB` is more appropriate. Let's change it!

> we also have other "Getting Started" docs 
Which ones?
Maybe, we need to rename them appropriately.

Reply:
@feeblefakie Sorry, let me clarify. The other docs I was referring to already have "Getting Started ..." in their titles (for example, [Getting Started with ScalarDB Analytics with PostgreSQL](https://scalardb.scalar-labs.com/docs/latest/scalardb-analytics-postgresql/getting-started/)). 

In the future, if we decide to not use "Getting Started" for this "Getting Started with ScalarDB" doc, we should keep in mind our other docs that have "Getting Started" in the title (like the one above) so that we can maintain consistency.

Reply:
@choplin In addition to the configuration section, the page also contains the following sections:

Create and load the database schema:
https://github.com/scalar-labs/scalardb/blob/c089fe1b0219db53b0daa3a6b0b0d4149d52e258/docs/getting-started-with-scalardb.md#create-and-load-the-database-schema

Execute transactions and retrieve data:
https://github.com/scalar-labs/scalardb/blob/c089fe1b0219db53b0daa3a6b0b0d4149d52e258/docs/getting-started-with-scalardb.md#execute-transactions-and-retrieve-data

Do you still consider this to be "too general"?

Reply:
Ah, sorry that I misunderstood what page we are talking. I said "too general" for the pages mentioned in the description of this PR, like https://scalardb.scalar-labs.com/docs/latest/getting-started-with-scalardb-on-cassandra/.

Reply:
@choplin Understood. So let's keep this page title with `Getting Started with ScalarDB`. Thank you!